### PR TITLE
Fix init process

### PR DESCRIPTION
**Pull request by damienbod**
* fixing init process, using APP_INITIALIZER, and proper support for angular guards
* removed override_well_known_configuration, well_known_configuration now loaded from the APP_INITIALIZER
* removed override_well_known_configuration_url, well_known_configuration now loaded from the APP_INITIALIZER

### CODE REVIEW

File: API_DOCUMENTATION.md [hunk 1]
```diff
@@ -166,45 +154,82 @@ Example using:
...
+            authWellKnownEndpoints.setWellKnownEndpoints(this.oidcConfigService.wellKnownEndpoints);
+
+            this.oidcSecurityService.setupModule(openIDImplicitFlowConfiguration, authWellKnownEndpoints);
+
+        });
+
+        console.log('APP STARTING');
+    }
 }
 ```
```
Comment (robisim74) [inline API_DOCUMENTATION.md]: l10nLoader?

Comment (damienbod) [reply API_DOCUMENTATION.md]: thanks, removed

**Comment by robisim74** [comment]
@damienbod Does it work with and without APP_INITIALIZER?

File: README.md [hunk 2]
```diff
@@ -232,6 +232,55 @@ You can add any configurations to this json, as long as the stsServer is present
...
+            this.oidcSecurityService.setupModule(openIDImplicitFlowConfiguration, authWellKnownEndpoints);
+    }
+}
+
+
+```
+
 <strong>Using Guards</strong>
 
 ```TypeScript
```
Comment (robisim74) [inline README.md]: Great!

**Comment by profet23** [comment]
w00t. Excited for this to hit master.

**Pull request merged**